showkase-browser-testing/build.gradle

```diff
@@ -43,20 +42,24 @@ android {
         exclude 'META-INF/AL2.0'
         exclude 'META-INF/LGPL2.1'
     }
+    configurations {
+        all {
```

Comment:
add a comment around why we are adding this otherwise there's no way we will remember it (like I've forgotten 20 other things throughout this codebase 😅)